coverage/src/BamView-WebComponent.js

```diff
@@ -176,9 +193,10 @@ class BamViewChart extends HTMLElement {
             if (resizeTimeout) clearTimeout(resizeTimeout);
             resizeTimeout = setTimeout(() => {
                 entries.forEach(entry => {
-                    if (entry.target === this.bamViewContainer) {
-                        this.bamViewContainer.innerHTML = ''; // Clear the current SVG
-                        this._bamView = createBamView(this.validBamHeader, this.validBamReadDepth, this.bamViewContainer, this.broker);
+                    if (entry.target === this.bamViewContainer ) {
+                        if (this.bamReadDepth && this.bamHeader) {
```

Comment:
This doesn't need to be a nested if. You can put all the conditions in a single statement